Formula/f3d.rb

```diff
@@ -0,0 +1,39 @@
+class F3d < Formula
+  desc "Fast and minimalist 3D viewer"
+  homepage "https://kitware.github.io/F3D/"
+  url "https://gitlab.kitware.com/f3d/f3d/-/archive/v1.0.1/f3d-v1.0.1.tar.gz"
+  sha256 "fb362dba3ccf49db9e8841d8a5310f37399bfea8866b6e040ce85670d54b97f7"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "vtk"
+
+  def install
+    # Is fixed upstream, remove this line for v1.1
+    inreplace "src/F3DOptions.cxx", "/etc/f3d/", "/usr/local/etc/f3d/"
+
+    args = std_cmake_args + %W[
+      -DCMAKE_BUILD_TYPE:STRING=Release
```

Comment:
Specifying a `Release` build is already part of `std_cmake_args`.

Reply:
thanks!